.github/workflows/python-app.yml

```diff
@@ -0,0 +1,36 @@
+name: Python application
+
+on:
+  push:
```

Comment:
Please specify the folder here so as to not be disturbed by the frontend or general changes

Reply:
Fixed